dashboard/app/jobs/evaluate_rubric_job.rb

```diff
@@ -212,10 +231,10 @@ def self.log_metrics(response)
     rubric_ai_evaluation.save!
   end
 
-  RETRIES_ON_RATE_LIMIT = 3
+  ATTEMPTS_ON_RATE_LIMIT = 3
 
-  # Retry on any reported rate limit (429 status) 'exponentially_longer' waits 3s, 18s, and then 83s.
-  retry_on TooManyRequestsError, wait: :exponentially_longer, attempts: RETRIES_ON_RATE_LIMIT do |job, error|
+  # Retry on any reported rate limit (429 status). With 3 attempts, 'exponentially_longer' waits 3s, then 18s.
+  retry_on TooManyRequestsError, wait: :exponentially_longer, attempts: ATTEMPTS_ON_RATE_LIMIT do |job, error|
```

Comment:
I'm not fully understanding why we're notifying Honeybadger for rate limit but not notifying Honeybadger for the other errors.

Reply:
this was an oversight -- if there's agreement on the overall approach, I'll update the rate limit handling to be more like the other ones I updated in this PR.